src/tools/cephfs/cephfs-shell

```diff
@@ -1117,23 +1117,36 @@ sub-directories, files')
         """
         Disk Usage of a Directory
         """
-        if args.dirs == '':
-            args.dirs = cephfs.getcwd().decode('utf-8')
-        for dir_ in args.dirs:
-            if args.r:
-                for i in reversed(sorted(set(dirwalk(dir_)))):
-                    i = os.path.normpath(i)
+        dir_passed = args.dirs
+        if args.dirs is '':
+            dir_passed = cephfs.getcwd().decode('utf-8')
+            if len(dir_passed) > 1:
+                dir_passed = dir_passed.split()
+
+        for dir_ in dir_passed:
+                dir_trav = sorted(set(dirwalk(dir_)))
```

Comment:
It would be nice if you can indent the block inside this `for` with 4 spaces (like the rest of code in here) instead of 8.

Reply:
I have fixed it in the new patch.